test/parallel/test-buffer-slice.js

```diff
@@ -83,12 +83,12 @@ assert.strictEqual(Buffer.alloc(0).slice(0, 1).length, 0);
 
 {
   // Single argument slice
-  assert.strictEqual('bcde',
-                     Buffer.from('abcde', 'utf8').slice(1).toString('utf8'));
+  assert.strictEqual(Buffer.from('abcde', 'utf8').slice(1).toString('utf8'),
+      'bcde',);
```

Comment:
@KaskMartin - linter failed for this change, basically at this line. The error message is:
`message: Expected indentation of 21 spaces but found 6.`
can you fix it and push again? basically you need to move this line 21 spaces ahead, instead of 6. thanks.

Reply:
Will do